src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDurationMappingTest.java

```diff
@@ -0,0 +1,371 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.mockito.Mock;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.reference.data.model.HearingDuration;
+import uk.gov.hmcts.reform.sscs.reference.data.service.HearingDurationsService;
+import uk.gov.hmcts.reform.sscs.service.holder.ReferenceDataServiceHolder;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+
+public class HearingsDurationMappingTest  extends HearingsMappingBase {
+    @Mock
+    private HearingDurationsService hearingDurations;
+
+    @Mock
+    private ReferenceDataServiceHolder referenceDataServiceHolder;
+
+    @DisplayName("when a valid adjournCaseDuration and adjournCaseDurationUnits is given getHearingDuration returns the correct duration Parameterized Tests")
+    @ParameterizedTest
+    @CsvSource(value = {
+        "120,minutes,120",
+        "1,sessions,165",
+    }, nullValues = {"null"})
+    void getHearingDuration(String adjournCaseDuration, String adjournCaseDurationUnits, int expected) {
+
+        SscsCaseData caseData = SscsCaseData.builder()
```

Comment:
caseData is setup in the same way several times. It would be better to extract a helper function to avoid the repetition.